medical/security/medical_security.xml

```diff
@@ -1,6 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
-<openerp>
-<data noupdate="0">
+<odoo noupdate="0">
 
     <record id="group_medical_user" model="res.groups">
```

Comment:
These groups should be presented in a selection for the `Medical` type, instead of as independent check boxes